client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -180,12 +187,29 @@ export const RequestLineEdit = ({
         </Box>
         <Box>
           {/* Right column content */}
+          <Box display="flex" flexDirection="row">
+            {isPacksEnabled && (
+              <Box display="flex" justifyContent="flex-end" alignItems="center">
```

Comment:
```suggestion
          <Box
            display="flex"
            flexDirection="row"
            justifyContent="flex-end"
            paddingBottom={1}
            paddingRight={2.5}
          >
            {isPacksEnabled && (
              <Box display="flex">
```

To get the switch aligning with the input (: + add padding so they're not so close together~

Reply:
Ah nice that is great! I thought it looked a bit odd. Thank you